src/waveform/renderers/waveformrenderersignalbase.cpp

```diff
@@ -169,11 +169,14 @@ void WaveformRendererSignalBase::setup(const QDomNode& node,
     onSetup(node);
 }
 
-void WaveformRendererSignalBase::getGains(float* pAllGain, float* pLowGain,
-                                          float* pMidGain, float* pHighGain) {
+void WaveformRendererSignalBase::getGains(float* pAllGain,
+        bool applyCompensation,
```

Comment:
maybe reorder and make it a default to `false`?

Reply:
I prefer to keep it explicit and to keep it next to allGain.

And eventually, all this should be cleaned up and we should have a discussion about what waveforms we want to keep and how they should scale. This whole compensation business is quite weird.

Reply:
I agree. I'm a fan of explicitness as well. The idea was just to use the default value to suggest what direction we'd like to go to in the future.